src/main/java/org/prebid/server/settings/model/ParsedStoredDataResult.java

```diff
@@ -0,0 +1,19 @@
+package org.prebid.server.settings.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+import java.util.List;
+import java.util.Map;
+
+@AllArgsConstructor(staticName = "of")
+@Value
+public class ParsedStoredDataResult<S, I> {
+
+    S storedData;
```

Comment:
We can refactor a bit of `org.prebid.server.settings.model.StoredDataResult#storedIdToRequest` field to be generic type.
Then `ParsedStoredDataResult` model can be removed.